Library/ViewModels/CommentRepliesViewModelTests.swift

```diff
@@ -208,13 +215,17 @@ internal final class CommentRepliesViewModelTests: TestCase {
         inputAreaBecomeFirstResponder: false
       )
 
+      self.vm.inputs.viewDidLoad()
+
       self.loadRepliesAndProjectIntoDataSourceProject.assertValues([])
       self.loadRepliesAndProjectIntoDataSourceReplies.assertValues([])
+      self.loadRepliesAndProjectIntoDataSourceTotalCount.assertValues([])
 
-      self.vm.inputs.viewDidLoad()
+      self.scheduler.advance()
 
       self.loadRepliesAndProjectIntoDataSourceProject.assertValues([project])
       self.loadRepliesAndProjectIntoDataSourceReplies.assertValues([envelope.replies])
+      self.loadRepliesAndProjectIntoDataSourceTotalCount.assertValues([envelope.totalCount])
```

Comment:
Maybe update test name to reflect the `totalCount` being checked.

Reply:
If it's not too much trouble, maybe we could test pagination here as well?

There are two tests in `CommentsViewModel` that do this pretty thoroughly.
`testUpdatePagination_WhenLimitReached_CommentsAreLoadedIntoDataSource`
`testProjectPagination_WhenLimitReached_CommentsAreUpdatedInDataSource`

If you copy paste them in here, and just check the relevant stuff like the `replies`, `total` and `project` would give us further confidence in this piece of the feature.

Reply:
We now handle the pagination in the data source as opposed to the view model so I don't think I can look at the previous view model tests to test this. It's proving to be a bit tricky but I'll take a look at it for a bit longer.

Reply:
Extended this test to include tests for pagination.